pandas/tests/frame/test_apply.py

```diff
@@ -1056,3 +1056,37 @@ def test_non_callable_aggregates(self):
         expected = df.size
 
         assert result == expected
+
+    @pytest.mark.parametrize("test_input", [
+        # is either frame or tuple (frame, args, kwargs),
+        pd.DataFrame([[1, 2], [3, 4]]),
+        pd.DataFrame([[np.nan, 2], [3, 4]]),
+        pd.DataFrame(),
+    ])
+    @pytest.mark.parametrize(
```

Comment:
Hmm wonder if this would be better as a shared fixture - thoughts?

Reply:
yes pls make this a fixture (in conftest.py)